docs/QbeastFormat0.6.0.md

```diff
@@ -0,0 +1,85 @@
+# QbeastFormat 0.6.0
+
+**The version 0.6.0 introduces a new format for the Qbeast tables.** 
```

Comment:
Are there any compatibility implications to tables created with earlier versions that we need to highlight here?

Reply:
Yep, they are explained in the same document, alongside a set of steps to transform early datasets.